Webapp/sources/templates/sketcher_reload.html

```diff
@@ -39,6 +39,13 @@ <h4>{{ reaction.reaction_id }}</h4>
         <p>Please sketch or upload your reaction to begin. Click <a target="_blank" href="{{ url_for('main.marvin_js_help') }}">here</a> to view our Marvin JS help guide.</p>
         <p>Please note compounds drawn above or underneath the arrow will not be recognised.</p>
     </div>
+    <div>
+        <form action="{{ url_for('reaction_table.process') }}" method="post">
+            <label> Polymer Mode?
+                <input type="checkbox" id="polymer-mode-select" name="polymer-mode-select" onclick="savePolymerMode()" {% if reaction.polymer_mode %} checked {% endif %}>
```

Comment:
does this need changing to reflect the new db migration?

```suggestion
                <input type="checkbox" id="polymer-mode-select" name="polymer-mode-select" onclick="savePolymerMode()" {% if reaction.reaction_type %} checked {% endif %}>
```